imports/ui/pages/components/UserManagement/ProfileDetailsCard.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import { Button, Card, Input, notification } from "antd";
+import { compose } from "redux";
+
+import { translate } from "../../../HOCs/translate";
+import { EMAIL_PROPERTY, USERNAME_PROPERTY } from "../../../../constants/systemConstants";
+import injectSheet from "react-jss";
+import { dynamicUserProfileStyles } from "./dynamicUserProfileStyles";
+import { withTracker } from "meteor/react-meteor-data";
+import { mongoCss } from "../../../../api/client/collections";
+
+class ProfileDetailsCard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      username: "",
+      email: ""
+    };
+  }
+
+  handleInputChange = property => event => {
+    this.setState({ [property]: event.target.value });
+  };
+
+  updateUsernameAndEmail = () => {
```

Comment:
`handleUpdate`
In the future more fields can be added to this card so current name will not be accurate.